### PR TITLE
Again add a hyperlink to the Spring Cloud Readme file Java Development Kit(JDK)

### PR DESCRIPTION
**Pull request by ZhuXiaoBing-cn**
update #15505

### CODE REVIEW

**Review by chenrujun — APPROVED** [review]
LGTM.

**Review by zhoufenqin — APPROVED** [review]

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ Please follow [instructions here][contributing_md] to build from source or contr
 [service_bus_topic_binder]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder
 [spring_cloud_stream_binder_service_bus_multiple_binders]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-servicebus-queue-multibinders-sample
 [spring_cloud_stream_binder_service_bus_queue]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-servicebus-queue-binder-sample
-[spring_cloud_stream_binder_service_bus_topic]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-servicebus-topic-binder-sample
+[spring_cloud_stream_binder_service_bus_topic]: https://ghithub.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-servicebus-topic-binder-sample
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md:160]: Is this an unintentional change?

**Review by saragluna — CHANGES_REQUESTED** [review]
Thanks for adding these links. Overall is great, but one unintentional change is included. Let's change it and get this PR merged.

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md (hunk 1 above)
Comment (ZhuXiaoBing-cn) [reply sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md:160]: yes,I will change back

**Review by saragluna — APPROVED** [review]

**Pull request merged**